scripts/autotracking.lua

```diff
@@ -501,7 +503,7 @@ function updateLocationsFromMemorySegmentCorridor(segment)
         updateSectionMultipleChestCountFromByteAndFlag(segment, "@Cardia Grassy/Chests", { [0x628C] = 0x04, [0x628D] = 0x04, [0x628E] = 0x04 })
         updateSectionMultipleChestCountFromByteAndFlag(segment, "@Cardia Swampy/Chests", { [0x6289] = 0x04, [0x628A] = 0x04, [0x628B] = 0x04 })
         updateSectionMultipleChestCountFromByteAndFlag(segment, "@Cardia Forest/Chests", { [0x6284] = 0x04, [0x6285] = 0x04, [0x6286] = 0x04, [0x6288] = 0x04, [0x628F] = 0x04, [0x6290] = 0x04 })
-		updateSectionMultipleChestCountFromByteAndFlag(segment, "@Cardia Forest/Incentive", 0x6287, 0x04)
+		updateSectionSingleChestCountFromByteAndFlag(segment, "@Cardia Forest/Cardia Incentive", 0x6287, 0x04)
```

Comment:
Looks like the spacing is funny here

Reply:
Damnable thing used actual tabs as opposed to matching to space format like it bloody well should have. Fixing that up now. Will be addressing it (and the below) in the next merge.